package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "test": "cypress run --config video=false",
+    "ci": "yarn dev & wait-on http://localhost:3000 && yarn test"
```

Comment:
`&` is discouraged since it leaves zombie processes around; prefer `concurrently` or `wait-on`; available as npm packages